script/env.ps1

```diff
@@ -0,0 +1,58 @@
+# Install base dependencies from the script/base.sh file
+
+# Install openssl for certificate generation
+choco install -y openssl
+
+# Install playwright tool for integration tests on browsers
+# Ensure npm is added to PATH
+if (-not ([Environment]::GetEnvironmentVariable("Path", [EnvironmentVariableTarget]::Machine) -like "*npm*")) {
+    Write-Host "Adding npm to PATH..."
+    [Environment]::SetEnvironmentVariable("Path", "$env:Path;$(npm prefix -g)\node_modules\.bin", [EnvironmentVariableTarget]::Machine)
+}
+
+npm install -g playwright
+
+# Installing required python packages
+choco install -y python3
+
+# Ensure Python scripts directory is added to PATH
+if (-not ([Environment]::GetEnvironmentVariable("Path", [EnvironmentVariableTarget]::Machine) -like "*Python3*")) {
+    Write-Host "Adding Python scripts to PATH..."
+    [Environment]::SetEnvironmentVariable("Path", "$env:Path;$(Get-Command python3 | Select-Object -ExpandProperty Directory)", [EnvironmentVariableTarget]::Machine)
+}
+
+# Install mkdocs
+choco install -y mkdocs
+
+# Ensure Ruby scripts directory is added to PATH
```

Comment:
Error:

```log
Installing collected packages: watchdog, six, pyyaml, platformdirs, pathspec, packaging, mergedeep, markupsafe, markdown, colorama, pyyaml-env-tag, python-dateutil, jinja2, click, ghp-import, mkdocs
Successfully installed click-8.1.7 colorama-0.4.6 ghp-import-2.1.0 jinja2-3.1.3 markdown-3.6 markupsafe-2.1.5 mergedeep-1.3.4 mkdocs-1.5.3 packaging-24.0 pathspec-0.12.1 platformdirs-4.2.0 python-dateutil-2.9.0.post0 pyyaml-6.0.1 pyyaml-env-tag-0.1 six-1.16.0 watchdog-4.0.0
 The install of mkdocs was successful.
  Software install location not explicitly set, it could be in package or
  default install location of installer.

Chocolatey installed 2/2 packages.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
Adding Ruby scripts to PATH...
Get-Command : The term 'ruby' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At C:\Users\dtaas-windows\Downloads\DTaaS\script\env.ps1:30 char:64
+ ... :SetEnvironmentVariable("Path", "$env:Path;$(Get-Command ruby | Selec ...
+                                                  ~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (ruby:String) [Get-Command], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException,Microsoft.PowerShell.Commands.GetCommandCommand
```

Reply:
> Error:
> 
> ```
> Installing collected packages: watchdog, six, pyyaml, platformdirs, pathspec, packaging, mergedeep, markupsafe, markdown, colorama, pyyaml-env-tag, python-dateutil, jinja2, click, ghp-import, mkdocs
> Successfully installed click-8.1.7 colorama-0.4.6 ghp-import-2.1.0 jinja2-3.1.3 markdown-3.6 markupsafe-2.1.5 mergedeep-1.3.4 mkdocs-1.5.3 packaging-24.0 pathspec-0.12.1 platformdirs-4.2.0 python-dateutil-2.9.0.post0 pyyaml-6.0.1 pyyaml-env-tag-0.1 six-1.16.0 watchdog-4.0.0
>  The install of mkdocs was successful.
>   Software install location not explicitly set, it could be in package or
>   default install location of installer.
> 
> Chocolatey installed 2/2 packages.
>  See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
> Adding Ruby scripts to PATH...
> Get-Command : The term 'ruby' is not recognized as the name of a cmdlet, function, script file, or operable program.
> Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
> At C:\Users\dtaas-windows\Downloads\DTaaS\script\env.ps1:30 char:64
> + ... :SetEnvironmentVariable("Path", "$env:Path;$(Get-Command ruby | Selec ...
> +                                                  ~~~~~~~~~~~~~~~~
>     + CategoryInfo          : ObjectNotFound: (ruby:String) [Get-Command], CommandNotFoundException
>     + FullyQualifiedErrorId : CommandNotFoundException,Microsoft.PowerShell.Commands.GetCommandCommand
> ```

ruby is needed for this